pkg/controller/config/config_controller.go

```diff
@@ -313,18 +342,53 @@ func (r *ReconcileConfig) applyAddons(req reconcile.Request, cfg *op.Config) (re
 			Version: flag.TektonVersion})
 		return reconcile.Result{}, err
 	}
-	log.Info("successfully applied all resources")
+	log.Info("successfully applied all addon resources")
+
+	err := r.updateStatus(cfg, op.ConfigCondition{Code: op.AppliedAddons, Version: flag.TektonVersion})
+	return reconcile.Result{Requeue: true}, err
+}
+
+func (r *ReconcileConfig) applyNonRedHatResources(req reconcile.Request, cfg *op.Config) (reconcile.Result, error) {
+	log := requestLogger(req, "apply-non-redhat-resources")
+
+	//add TaskProviderType label to ClusterTasks (community, redhat, certified)
+	addnTfrms := []mf.Transformer{
+		// replace kind: Task, with kind: ClusterTask
+		transform.ReplaceKind("Task", "ClusterTask"),
+		transform.InjectLabel(flag.LabelTaskProviderType, flag.TaskProviderTypeCommunity, true),
```

Comment:
true is a bit misleading .. it is better to use an enum `overwrite | retain `

Reply:
done